src/css/custom.scss

```diff
@@ -358,18 +358,13 @@ body,
     color 0.5s 0.2s ease;
   z-index: 2;
 }
-.markdown details {
-  background: rgba(255, 255, 255, 0);
-  border-radius: 0.5rem;
-  backdrop-filter: blur(7px);
-  box-shadow: 0 0 2px rgba(0, 0, 0, 0.6);
-  padding: 0;
-  transition: padding 0.3s 0s ease-in, background 0.3s 0.4s ease-in;
-}
-.markdown details[open] {
-  background: rgba(255, 255, 255, 0.08);
-  padding: 15px;
-}
+// .markdown details {
```

Comment:
Is there any reason these comments are left in?

Reply:
Habit mainly. :smile: I'll remove on the next round of updates.